### ISSUE TITLE
Change plugin manager in neovim

### ISSUE THREAD

**Issue by alexanderjeurissen** [alexanderjeurissen/dotfiles#9]
Currently using dein but not really content with it.

*drawbacks*:
1. Progress indicator of installing plugins is less clear then vimplug.
2. the necessity to recache the runtimepath.
3. too many options for lazy loading, whilst plenty of plugins already have that in place.
4. Uninstalling plugins sometimes requires multiple recach attempts / vimrc sourcing, and leaves empty folders in the file system.
5. documentation is less clear then vimplug / pathogen.

Basically since I'm cutting down on plugin usage, removing anything I don't use often enough or where the added value compared to default vim is non present, the use for a complex plugin manager is not there..

*potential alternatives*
1. vim-plug
2. pathogen

**Comment by alexanderjeurissen** [comment]
reading / reference material: 

- https://www.reddit.com/r/neovim/comments/5npmcm/which_plugin_manager_should_i_use/
- https://bitbucket.org/vimcommunity/vim-pi
- https://github.com/neovim/neovim/issues/247
- https://medium.com/@sdboyer/so-you-want-to-write-a-package-manager-4ae9c17d9527

**Comment by alexanderjeurissen** [comment]
I ended up deciding on vimplug because it seems to be the most popular plugin manager at the moment in the neovim community. Also I feel that it strikes the right balance between feature rich, and simplicity / out of the box config.

see: #10
